go.mod

```diff
@@ -18,8 +18,8 @@ require (
 	github.com/gogo/protobuf v1.3.1
 	github.com/google/gofuzz v1.1.0
 	github.com/ipfs/go-log v1.0.4
-	github.com/keep-network/keep-common v1.2.1-0.20201117163745-878626866ba7
-	github.com/keep-network/keep-core v1.3.2-0.20201117170744-88a808971677
+	github.com/keep-network/keep-common v1.2.1-0.20201125164243-e39a6818bc8a
+	github.com/keep-network/keep-core v1.3.2-0.20201125171123-4549ea2b23dd
```

Comment:
Having https://github.com/keep-network/keep-core/pull/2180 merged now, let's switch this to some commit from `master`.